src/functionaltests/java/com/ericsson/ei/rabbitmq/connection/RabbitMQTestConnectionSteps.java

```diff
@@ -56,10 +56,11 @@
     private String rabbitMQPort;
 
     private static final Logger LOGGER = LoggerFactory.getLogger(RabbitMQTestConnectionSteps.class);
-    private static final String EIFFEL_EVENTS = "src/functionaltests/resources/eiffel_events_for_thread_testing.json";
+    private static final String EIFFEL_EVENTS = "src/functionaltests/resources/eiffel_events_for_test.json";
```

Comment:
Have you checked the differences between these files? If it is just a naming thing or any changes in json itself?

Reply:
> Have you checked the differences between these files? If it is just a naming thing or any changes in json itself?

Yes. Instead of testing with 4 events, here we have tested with only 1 event. So, only 1 event is being declared.